app/main.py

```diff
@@ -1 +1,59 @@
-#  write your code here
+from typing import Any
+
+
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> str:
```

Comment:
The method `print_name` is defined to return a `str`, but it actually returns `None` because it only prints a message. Consider changing the return type to `None`.